packages/graphback-datasync/src/providers/DataSyncConflictProvider.ts

```diff
@@ -3,7 +3,7 @@ import { DataSyncModelConflictConfig, DataSyncFieldNames, ConflictMetadata, Conf
 import { MongoDeltaSource } from '../deltaSource';
 import { DataSyncMongoDBDataProvider } from './DatasyncMongoDBDataProvider';
 
-export const MAX_RETRIES = process.env.CONFLICT_RESOLUTION_MAX_RETRIES || 3;
+export const MAX_RETRIES = parseInt(process.env.CONFLICT_RESOLUTION_MAX_RETRIES, 2) || 3;
```

Comment:
```suggestion
export const MAX_RETRIES = parseInt(process.env.CONFLICT_RESOLUTION_MAX_RETRIES, 10) || 3;
```

Why the base is 2 when parsing?

Reply:
Sorry I got confused :sweat_smile: